01-cloudformation/README.md

```diff
@@ -160,19 +160,34 @@ name.
 
 _Why do we prefer the YAML format for CFN templates?_
 
+#### The intrinsic functions
+
 #### Question: Protecting Resources
 
 _What else can you do to prevent resources in a stack from being deleted?_
 
 See [DeletionPolicy](https://aws.amazon.com/premiumsupport/knowledge-center/cloudformation-accidental-updates/).
 
+#### Use IAM to restrict users and utilize the DeletionPolicy attribute
```

Comment:
Other than using Termination Protection, to prevent resources from deletion, you can:
- Set the DeletionPolicy attribute to prevent the deletion of an individual resource at the stack level.
- Use IAM policies to restrict the ability of users to delete or update a stack and its resources.
- Assign a stack policy to prevent updates to stack resources.